data/hai/hai outfits.txt

```diff
@@ -107,58 +107,67 @@ effect "ion impact"
 
 outfit "Railgun Slug"
 	category "Ammunition"
-	cost 60
+	cost 13
 	thumbnail "outfit/railslug"
-	"mass" 0.02
+	"mass" 0.003
 	"railgun slug capacity" -1
-	description "These railgun slugs incorporate energy-discharging capacitors that deal a small amount of hull damage even when the target's shields are functioning, and they can punch crippling holes in the hull of a ship whose shields are down."
+	description "These railgun slugs incorporate energy-discharging capacitors that deal a small amount of ion damage to their target. Although not terribly damaging, this ion damage combined with the significant recoil that each shot has turns Flea drones into effective long-range support ships."
+
+outfit "Railgun Slug Rack"
+	category "Ammunition"
+	cost 15000
+	thumbnail "outfit/railslug rack"
+	"mass" 1
+	"outfit space" -3
+	"railgun slug capacity" 500
+	ammo "Railgun Slug"
+	description "The railgun slug rack is used to store and load extra railgun ammunition."
 
 outfit "Railgun"
 	category "Secondary Weapons"
 	cost 20000
 	thumbnail "outfit/railgun"
-	"mass" 4
-	"outfit space" -7
-	"weapon capacity" -7
+	"mass" 1
+	"outfit space" -4
+	"weapon capacity" -4
 	"gun ports" -1
-	"railgun slug capacity" 150
+	"railgun slug capacity" 1000
 	weapon
 		sprite "projectile/rail slug"
 			"frame rate" 30
 		ammo "Railgun Slug"
 		icon "icon/rail gun"
 		sound "ion impact"
-		"fire effect" "rail sparks" 2
-		"live effect" "rail sparks" 5
+		"fire effect" "rail sparks"
 		"hit effect" "bullet impact"
 		"inaccuracy" 3
-		"velocity" 22
+		"velocity" 44
 		"lifetime" 1
-		"reload" 120
-		"firing energy" 92
-		"firing heat" 39
-		"hit force" 44
+		"range override" 2100
+		"reload" 12
+		"firing energy" 3.5
+		"firing heat" 7.2
+		"firing force" 70
 		"stream"
 		"submunition" "rslug" 1
-	description "The Hai created this compact weapon for their Flea drones, providing mobile long-range support for the close quarters Pond Strider carrier."
+	description "The Hai designed this weapon to best compliment their Flea drones. Each railgun provides a significant amount of force upon firing, pushing the Flea drone backwards and therefore forcing it to keep its distance from its target."
```

Comment:
Hello team. Just leaving a note: pretty sure this should be 'to best complement their Flea drones'.